oc.spec

```diff
@@ -28,51 +35,56 @@
 %global make %{os_git_vars} && make SOURCE_GIT_TAG:="${OS_GIT_VERSION}" SOURCE_GIT_COMMIT:="${OS_GIT_COMMIT}" SOURCE_GIT_MAJOR:="${OS_GIT_MAJOR}" SOURCE_GIT_MINOR:="${OS_GIT_MINOR}" SOURCE_GIT_TREE_STATE:="${OS_GIT_TREE_STATE}"
 %endif
 
-%if ! 0%{?local_build:1}
-Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
-%endif
-Name:           openshift-clients
+Name:           %{package_name}-clients
 Version:        %{version}
 Release:        %{release}%{dist}
 Summary:        OpenShift client binaries
 License:        ASL 2.0
 URL:            https://%{import_path}
 
+%if ! 0%{?local_build}
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+%endif
+
 # If go_arches not defined fall through to implicit golang archs
 %if 0%{?go_arches:1}
 ExclusiveArch:  %{go_arches}
 %else
 ExclusiveArch:  x86_64 aarch64 ppc64le s390x
 %endif
 
-#BuildRequires:  bsdtar
 BuildRequires:  golang >= %{golang_version}
 BuildRequires:  krb5-devel
 BuildRequires:  rsync
 
-Provides:       atomic-openshift-clients
-Obsoletes:      atomic-openshift-clients
+Provides:       atomic-openshift-clients = %{version}
+Obsoletes:      atomic-openshift-clients = %{version}
 Requires:       bash-completion
 
+%if 0%{?ose_build}
+Conflicts:      okd-clients
+%else
+Conflicts:      openshift-clients
+%endif
+
 %description
 %{summary}
 
 %package redistributable
-Summary:        OpenShift Client binaries for Linux, Mac OSX, and Windows
-Provides:       atomic-openshift-clients-redistributable
-Obsoletes:      atomic-openshift-clients-redistributable
+Summary:        %{product_name} Client binaries for Linux, Mac OSX, and Windows
```

Comment:
Hrm, technically no matter what this is still openshift - the binaries say openshift in them.  I don't think you need to change this, the same code works for all versions.